Microsoft.HealthVault.Fhir/Microsoft.HealthVault.Fhir.csproj

```diff
@@ -7,7 +7,13 @@
 
   <ItemGroup>
     <PackageReference Include="Hl7.Fhir.STU3" Version="0.93.4" />
-    <PackageReference Include="Microsoft.HealthVault" Version="1.65.20508.1-preview" />
+    <PackageReference Include="NodaTime" Version="2.0.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.HealthVault">
+      <HintPath>..\..\healthvault-dotnetstandard-sdk\Microsoft.HealthVault\bin\Debug\netstandard1.4\Microsoft.HealthVault.dll</HintPath>
+    </Reference>
```

Comment:
As discussed in email, the fastest path is getting those fixes into the s65 branch.  Going forward we'll get master into a state where we can push a -preview release from it at any time.

---
In reply to: [123657267](https://github.com/Microsoft/healthvault-fhir-library/pull/3#discussion_r123657267) [](ancestors = 123657267)